Add `Gulano Cup #5` tournament article (#12870)
ppy/osu-wiki: wiki/Tournaments/GNV/5/en.md

---
tags:
  - GNV5
  - Gulanocup
  - Gulanocap
  - Gulanova
---

# Gulano Cup #5

![Gulano Cup #5 banner](img/banner.png)

> **Gulano Cup #5** was a 1v1 double-elimination osu!mania 4K tournament hosted by ::{ flag=ID }:: [danar](https://osu.ppy.sh/users/11184912) and ::{ flag=ID }:: [Revv-](https://osu.ppy.sh/users/12424909). It was open exclusively to Indonesian players. It has a **wildcard** map gimmick that is submitted by players and can be used in the match by randomly picking from the wildcard map list. Only non-MWC players are able to play in this tournament. It was the fifth instalment of the Gulano Cup series.

Author: harusnya buat semua iterasi MWC, jadi nanti aku ganti ke link indexnya ya

> ## Tournament Schedule

Reviewer: ```suggestion
## Tournament schedule
```

| Programmer | ::{ flag=ID }:: [Antipole](https://osu.ppy.sh/users/17258072), ::{ flag=ID }:: [danar](https://osu.ppy.sh/users/11184912), ::{ flag=ID }:: [FelixSpade](https://osu.ppy.sh/users/2651304), ::{ flag=ID }:: [Nqinn](https://osu.ppy.sh/users/11312272) |
| Composer | ::{ flag=ID }:: Altair Aquila[^commission] ([YouTube](https://www.youtube.com/@AltairAquila)), ::{ flag=ID }:: Mariela Riona[^commission] ([YouTube](www.youtube.com/@MarielaRionaCh)), ::{ flag=ID }:: morethanahalf[^commission] ([SoundCloud](https://soundcloud.com/morethanahalf)), ::{ flag=ID }:: [Reyi](https://osu.ppy.sh/users/13385865), ::{ flag=ID }:: sarochi ([Spotify](https://open.spotify.com/artist/4zCVrst2LBGjrrlBKQhNoc)), ::{ flag=ID }:: typeMARS[^commission] ([SoundCloud](https://soundcloud.com/typemars)) |
| Sheeter | ::{ flag=ID }:: [danar](https://osu.ppy.sh/users/11184912), ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515), ::{ flag=ID }:: [Revv-](https://osu.ppy.sh/users/12424909) |
| Editor Wiki | ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515) |

> | Sponsor | [FTG]Kagamine, AsanagiYori, danar, Halsy, hira_osu, Ilham, lolohpela, MightyTech, Molten Cheese, typeMARS, Yuvya Anastasya |

Author: agak sulit soalnya donatenya di trakteer jadi ada yang nggak ada profile, aku samain biar lebih rapi aja

| #71 | ::{ flag=ID }:: [hafishoka](https://osu.ppy.sh/users/8071728) | 68.5 |
| #72 | ::{ flag=ID }:: [RealTBNRKenny](https://osu.ppy.sh/users/17731576) | 69.5 |
| #73 | ::{ flag=ID }:: [MGD FURRY](https://osu.ppy.sh/users/16800937) | 70.7 |
| #74 | ::{ flag=ID }:: [menprogemink](https://osu.ppy.sh/users/30547760) | 70.9 |
| #75 | ::{ flag=ID }:: [124fa](https://osu.ppy.sh/users/16263257) | 71.6 |
| #76 | ::{ flag=ID }:: [zReol](https://osu.ppy.sh/users/23539213) | 71.7 |
| #77 | ::{ flag=ID }:: [MyRisu](https://osu.ppy.sh/users/36244414) | 72.4 |
| #78 | ::{ flag=ID }:: [D31T7](https://osu.ppy.sh/users/7421690) | 77.3 |
| #79 | ::{ flag=ID }:: [KimoTaki](https://osu.ppy.sh/users/13451556) | 77.7 |
| #80 | ::{ flag=ID }:: [anteiku7](https://osu.ppy.sh/users/7013247) | 80 |

## Ruleset
These series of rules are enforced and used for Gulano Cup #5 so that there is no disruption if there is a problem. With their involvement, all people involved in Gulano Cup #5, including staff and players, are considered to know and understand all of these rules. These rules are enforced fairly based on the principles of logic and common sense by all Gulanova staff.

> ### Qualifier Procedure

Reviewer: ```suggestion
### Qualifier procedure
```

> - The referee will remind players about 15 minutes via discord before the scheduled lobby starts.

Reviewer: ```suggestion
- The referee will remind players about 15 minutes via Discord before the scheduled lobby starts.
```

- The lobby will start when all players have entered, or 10 minutes have passed from the scheduled time.

> - Players will play pool stages 1-6 in order. After all stages have been played, players are given a 2-minute break before playing stages 1-6 again (second chance). And this second chance is optional but recommended.

Reviewer: ```suggestion
- Players will have to play all six Qualifier stages in order. After all stages have been played, players will be given a 2-minute break before they go on their second Qualifier attempt. Playing this second run is not mandatory, but recommended.
```

- The best score from each stage will be taken as the qualifier score.

> ### Match Procedure

Reviewer: ```suggestion
### Match procedure
```

> - The referee will remind both players about 15 minutes via discord before the match starts.

Reviewer: ```suggestion
- The referee will remind both players about 15 minutes via Discord before the match starts.
```

> - Players are entitled to play one warm-up in the lobby (each player).

Reviewer: ```suggestion
- Players may play up to two warm-up maps in the lobby before the match (one from each player).
```

> - After the warm-up, both players will do a !roll. The highest roll has the right to choose to ban or pick first with the ABBA format. Example: If player A chooses the first pick, then player B will ban, followed by player A. then player A can choose a pick. The winner of the roll is guaranteed to get the first protection

Reviewer: ```suggestion
- After the warm-up, both players will have to do a !roll. The player with the higher roll number has the right to choose to ban or pick first following the ABBA format. For example, If player A chooses to pick first, then player B will get the first ban, followed by player A's ban and finally player A can choose a pick. The winner of the roll is guaranteed to get the first protect.
```

- Players will take turns choosing maps until a winner is determined. If both players only need 1 more point to win the match, then a Tiebreaker map will be played.

> ### General Rules

Reviewer: ```suggestion
### General rules
```

> - In general, all attendees follow the rules of osu! community rules that already include sportsmanship, not doing conditions, and general manners.

Reviewer: ```suggestion
- Players, staffs, and other attendees are required to abide to the [osu! community rules](/wiki/Rules).
```

- No external sensitive topics are allowed unless they are related to this event.
- Feedback is made constructively, without it considered as just a joke.

> - Streaming availability is prioritized for matches that have a schedule in advance, the later the reschedule the less likely it is to get the stream.

Reviewer: ```suggestion
- Matches will be streamed according to the streamers' availability, with matches that have their schedule locked in early being prioritised. While players may reschedule their matches to a more suitable time, the later the reschedule is agreed, the less likely it is for the match to be streamed.
```

> ### Warm-up Rules

Reviewer: ```suggestion
### Warm-up rules
```

> - Players are allowed to send warm-up maps before the match starts.

Reviewer: ```suggestion
- Players are allowed to send their own map of choice to the referee to be played as a warm-up map before the match starts.
```

> - Maps sent to the referee can be in the form of dm, in the room, discord, or anything whose message is conveyed to the referee.

Reviewer: ```suggestion
- Warm-up maps may be sent to the referee through DMs, Discord, directly in the room, or any other media without restriction.
```

> - Maximum warm-up duration is 3 minutes 30 seconds without negotiation.

Reviewer: ```suggestion
- Warm-up maps may not exceed 3 minutes and 30 seconds.
```

> - It is forbidden to use the entire mappool as a warm-up.

Reviewer: ```suggestion
- Maps that are present in the corresponding stage's mappool may not be picked as a warm-up map.
```

Author: kalo buat seluruh mappool wordingnya gimana ya?

> - Must be in 4k osu!mania mode.

Reviewer: ```suggestion
- Warm-up maps must be a 4K osu!mania map.
```

> ### Wildcard Rules

Reviewer: ```suggestion
### Wildcard rules
```

> - Each player is required to send 4 maps of their choice via the available google form with a maximum duration of 4 minutes or a minimum of 10 seconds and must be osu!mania 4K.

Reviewer: ```suggestion
- Each player is required to send 4 osu!mania 4K maps of their choice, all no shorter than 10 seconds and no longer than 4 minutes, through [this submission form](LINK).
```

Reviewer: itu link-nya nanti jangan lupa diisi ya btw wkwk

Author: perlu kah dikasih link? soalnya udah di close

Reviewer: iya, gpp kasih aja meskipun udah di-close juga

> - Maps that damage the client or cause crashes are not allowed.

Reviewer: ```suggestion
- Submitted maps may not be unplayable or otherwise causes the osu! client to crash.
```

- If one of the players disapproves to play using the wildcard gimmick, the wildcard mappool will be closed.
- Wildcards will be randomly selected based on the results of the google form submission if the player chooses from the mappool.
- Maps that have been randomly selected in the previous match cannot be used again for the next match.
- Each player only has 1 wildcard quota and cannot take the opponent's wildcard.
- The Wildcard system follows from the first batch, then followed by the next batch, which means that the later the batch, the less likely it is to be picked.
- If a player does not submit the wildcard to the form, the player is disqualified.

> ### WBD Rules

Reviewer: ```suggestion
### WBD rules
```

- Players will be considered resigning if there is no information beyond the scheduled 15 minutes.
- Players will get the right to negotiate if there is information about obstacles, but players who are present have the right to take Win by Default.
- If both are absent, the higher seed gets Win By Default. However, it is permissible to reschedule and cancel Win By Default up to a maximum of before the next showcase round.
- There is no WBD in the Grand Finals. The latest delay is 1 week and if it is still not possible then the Prizepool will be shifted to the players who are present by 80%. If no one plays at all, then the Prizepool will be canceled.

> ### Qualification Rules

Reviewer: ```suggestion
### Qualification rules
```

- All players must schedule before Saturday. If it is past Saturday, please contact the Referee, prioritized those who are active at that time.
- If a player is absent for more than 10 minutes in the scheduled lobby, the player is considered absent from the lobby and is required to confirm rescheduling with the Referee (if there is no response from the Referee then there is no schedule at that time).
- Players are disqualified if they do not play the qualifier until 1 hour before the Mappool Showcase.

> ### Match Rules

Reviewer: ```suggestion
### Match rules
```

- Scoring system uses Score v2, Head to Head (No Fail has no effect).
- Protected maps cannot be banned.
- Mods that are allowed: Mirror, Hidden, Fade In, and Flashlight.
- Each player has the right to call a 3 x 1 minute timeout.
- If the player does not choose a map for 1 minute, the timeout will be used.
- If the player is not ready after choosing a map, the timeout will be used.
- If the timeout is up and still has not chosen a map, then the pick is given to the opponent.
- If the timeout is up and still not ready after choosing a map, then the point is given to the opponent.
- If there is no response from the player for 10 minutes, then the opponent gets Win By Default.
- Players have the right to request an abort during the match before 30 seconds have passed, only 1 repetition is allowed (a different map will reset the abort).
- Abort timing is based on the referee's knowledge, evidence can be seen on irc or replay on Livestream (if available).

> - Everything can be negotiated with the referee and opponent if there is information about obstacles such as connection, power outages, or the likes of it.

Reviewer: ```suggestion
- Players are free to discuss upon obstacles such as connection issues, power outages, and the likes of it to the referee and their opponent at any time.
```

> - Especially for Grand Finals, players in the last match are required to showcase the Tiebreaker map after the match ends.

Reviewer: ```suggestion
- Exclusive for the Grand Finals, players participating in the Grand Final match are required to showcase the Tiebreaker map after the match ends.
```

> ### Reschedule Rules

Reviewer: ```suggestion
### Reschedule rules
```

> - Active players reschedule with a clear time along with the date and time of the start of their match.

Reviewer: ```suggestion
- Upon submitting a rescehdule, players are required to inform an exact date and time for their new match to the referee.
```

Author: rescehdule -> reschedule ya

Reviewer: iyap

> - Determination of reschedule in a round must have a day between Friday, Saturday, Sunday, or Monday 1 week after the mappool showcase of the round with a deadline of 16:00 on Monday.

Reviewer: ```suggestion
- The new match time must fall into a Friday, Saturday, Sunday, or Monday of their corresponding stage with a deadline of 16:00 on Monday.
```

| Programmer | ::{ flag=ID }:: [Antipole](https://osu.ppy.sh/users/17258072), ::{ flag=ID }:: [danar](https://osu.ppy.sh/users/11184912), ::{ flag=ID }:: [FelixSpade](https://osu.ppy.sh/users/2651304), ::{ flag=ID }:: [Nqinn](https://osu.ppy.sh/users/11312272) |
| Composer | ::{ flag=ID }:: Altair Aquila[^commission] ([YouTube](https://www.youtube.com/@AltairAquila)), ::{ flag=ID }:: Mariela Riona[^commission] ([YouTube](https://www.youtube.com/@MarielaRionaCh)), ::{ flag=ID }:: morethanahalf[^commission] ([SoundCloud](https://soundcloud.com/morethanahalf)), ::{ flag=ID }:: [Reyi](https://osu.ppy.sh/users/13385865), ::{ flag=ID }:: sarochi ([Spotify](https://open.spotify.com/artist/4zCVrst2LBGjrrlBKQhNoc)), ::{ flag=ID }:: typeMARS[^commission] ([SoundCloud](https://soundcloud.com/typemars)) |
| Spreadsheet manager | ::{ flag=ID }:: [danar](https://osu.ppy.sh/users/11184912), ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515), ::{ flag=ID }:: [Revv-](https://osu.ppy.sh/users/12424909) |
| Statistician | ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515) |
| Wiki editor | ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515) |

> | Sponsor | \[FTG\]Kagamine, AsanagiYori, danar, Halsy, hira_osu, Ilham, lolohpela, MightyTech, Molten Cheese, typeMARS, Yuvya Anastasya |

Reviewer 2: was not linking to sponsors' profiles a conscious decision?

Author: the sponsors come from twitch, private funding and saweria, any for some we can't find profile so decided to not link them

| #123939 | ::{ flag=ID }:: [vivol](https://osu.ppy.sh/users/31408326) |
| #12341933 | ::{ flag=ID }:: [Dogethedoge25](https://osu.ppy.sh/users/15597315) |

## Podium

This competition has come to an end and resulted in the following podium:

![Gulano Cup #5 podium](img/podium.jpg "Gulano Cup #5 Podium")

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "Juara pertama") | ::{ flag=ID }:: [lisachandra](https://osu.ppy.sh/users/26634482) |
| ![Silver crown](/wiki/shared/crown-silver.png "Juara kedua") | ::{ flag=ID }:: [[LS]Robi](https://osu.ppy.sh/users/21900872) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "Juara ketiga") | ::{ flag=ID }:: [MightyTech](https://osu.ppy.sh/users/34454156) |
| Non-podium finalists | ::{ flag=ID }:: [[LS] akarii](https://osu.ppy.sh/users/20040064), ::{ flag=ID }:: [DAENGXicisae-](https://osu.ppy.sh/users/8029883), ::{ flag=ID }:: [SnakesYxle](https://osu.ppy.sh/users/22842079) |

## Mappools

### Grand Finals

**[Download the mappack here! (91 MB)](https://drive.google.com/file/d/1jx6MHwSo7L2JDdpSgf8zTy3cGt0NLoKc/view)**\
[Watch the showcase VOD here!](https://www.youtube.com/watch?v=SjNkzGL41Zc)

- Rice

>   1. [AXiS as EQUiON - Protogene (Cut Ver.) [DOMiNANCE]](https://osu.ppy.sh/beatmapsets/2292615#mania/4895261)

Reviewer 3: should be `artist - title (creator) [difficulty]`. the map creator is missing from all of these

this has taken a long time already so i'll do this later when i have more time

Author: was thinking to not include the creator because the focused one is who submitted the map, and I figured it would be too long

but if it's needed I can add it

Author: oh nevermind I understand it now, I can do that for the mappool, but can I do what I said above for wildcard? Also one more thing, for collabs/edit diffs can we change them to the original mapper(s)?

Reviewer 3: this format exists elsewhere so it would be preferable to use the same map link format everywhere. example: https://osu.ppy.sh/wiki/en/Beatmap_Spotlights/Seasons/2023_Spring
for collabs you can put in mappers separated by commas. guest diff mappers are usually indicated in the diff name but if not i guess you can put the guest mapper as the creator

  42. [sabi - true DJ MAG top ranker's song Zenpen (katagiri Remix) [fn(top=true,sv=false)]](https://osu.ppy.sh/beatmapsets/1569250#mania/3204544) submitted by [Dogethedoge25](https://osu.ppy.sh/users/15597315)
  43. [DJ SHARPNEL - Kaze no Naka no Subaru ~Hyougenshatachi~ [subarushii]](https://osu.ppy.sh/beatmapsets/1628745#mania/3325116) submitted by [TheLegend3RD](https://osu.ppy.sh/users/14732769)
  44. [Sharks & Chime - Water Elemental [Stage 3: Stormbound]](https://osu.ppy.sh/beatmapsets/1917756#mania/3957466) submitted by [spectator_boi](https://osu.ppy.sh/users/17400577)
  45. [Risshuu feat. Choko - Take [Beyond]](https://osu.ppy.sh/beatmapsets/1377628#mania/2847100) submitted by [KafuuChlno](https://osu.ppy.sh/users/31102647)
  46. [korea - oronaminC [korean vibro]](https://osu.ppy.sh/beatmapsets/543648#mania/1152244) submitted by [RaffCo](https://osu.ppy.sh/users/7589503)
  47. [Angelic Burster - Star Bubble [AngeliChan!!]](https://osu.ppy.sh/beatmapsets/1271321#mania/2641914) submitted by [KyoukaRy](https://osu.ppy.sh/users/13403380)

>   48. Laser Imouto - Prismatix \[SMOCMouse \[Jumpstream+trill\] 1.1x (220bpm) OD8\] submitted by [danar mw](https://osu.ppy.sh/users/13859109)

Reviewer 3: map is missing https://osu.ppy.sh/b/3687529

  44. [Virus Syndicate & Virtual Riot & Dion Timmer - Gang Shit [Tech Shit]](https://osu.ppy.sh/beatmapsets/966210#mania/2022451) submitted by [spectator_boi](https://osu.ppy.sh/users/17400577)
  45. [goreshit - satori de pon! [4K MX]](https://osu.ppy.sh/beatmapsets/435015#mania/1026633) submitted by [KafuuChlno](https://osu.ppy.sh/users/31102647)
  46. [t+pazolite - Garakuta Doll Play [4K Challenge]](https://osu.ppy.sh/beatmapsets/451327#mania/968241) submitted by [RaffCo](https://osu.ppy.sh/users/7589503)
  47. [Porter Robinson & Madeon - Shelter [Allein Quad 1.2]](https://osu.ppy.sh/beatmapsets/1490961#mania/3056264) submitted by [KyoukaRy](https://osu.ppy.sh/users/13403380)
  48. [Nakiri Ayame - Good-bye sengen [Ojou]](https://osu.ppy.sh/beatmapsets/1603078#mania/3273717) submitted by [danar mw](https://osu.ppy.sh/users/13859109)
  49. [Remo Prototype (CV: Hanamori Yumiri) - Sendan Life (katagiri Bootleg) [Chord Life [135]]](https://osu.ppy.sh/beatmapsets/1170842#mania/2442473) submitted by [beethiven](https://osu.ppy.sh/users/18560270)

>   50. HoneyWorks feat.Kotoha - Ai ni Deai Koi wa Tsuzuku \[SMOCMouse \[Chordjack\] 1.6x (320bpm) OD8\] submitted by [ImOyZo](https://osu.ppy.sh/users/14875989)

Reviewer 3: map is missing https://osu.ppy.sh/b/4164787

- The win by default mechanic is not applicable in the Grand Finals. Both grand finalists are given the right to delay their Grand Final match for up to 1 week, and if no consensus is reached until then, the prize pool will be automatically given to the player who is present by 80%. If both of the grand finalists are not able to play at all, the prize pool will be cancelled entirely.

### Qualifier rules

- All players must schedule before Saturday. If it is past Saturday, please contact the referee, prioritising those who are active at that time.
- If a player is absent for more than 10 minutes in the scheduled lobby, the player is considered absent from the lobby and is required to confirm rescheduling with the referee (if there is no response from the referee then there is no schedule at that time).
- Players are disqualified if they do not play the qualifier until 1 hour before the mappool showcase.

### Match rules

- The scoring system uses Score v2, Head to Head (No Fail has no effect).
- Protected maps cannot be banned.
- Mods that are allowed: Mirror, Hidden, Fade In, and Flashlight.

> - Each player has the right to call a 3 x 1 minute timeout.

Reviewer 3: is this what is meant here?

```suggestion
- Each player has the right to call a 1-minute timeout 3 times.
```
